packages/flutter/test/widgets/text_test.dart

```diff
@@ -1662,29 +1662,48 @@ void main() {
   });
 
   testWidgets('Mouse hovering over selectable Text uses SystemMouseCursor.text', (WidgetTester tester) async {
```

Comment:
can you write a test that selectable textspan with recognizer will still show SystemMouseCursors.click when hover over the recognizer?

Reply:
done